Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -219,6 +221,17 @@ private void OnFocusChanged(bool focus)
 
         public ScreenOrientation screenOrientation => Screen.orientation;
 
+        public ScreenKeyboard screenKeyboard
+        {
+            get => m_ScreenKeyboard;
```

Comment:
This can have negative impact on performance, if application needs to allocate somekind of resources for the keyboard, the first time the keyboard will show up, you might suffer a stutter. It's better if such stutter would be a part of application initialization rather than happen during gameplay.